features/atomics-wait-async.yml

```
@@ -0,0 +1,6 @@
+name: Atomics.waitAsync
+description: "The `Atomics.waitAsync()` static method waits asynchronously on a shared memory location and returns a Promise."
```

I had a hard time understanding this one! AFAICT, contrary to the intro on MDN, this doesn't actually return a `Promise`, but it does return an object that _can_ contain a promise? Here's my suggestion to navigate that fact:

```suggestion
description: The `Atomics.waitAsync()` static method waits for a value in a shared memory location, providing a promise when the expected value is not yet in memory. The `waitAsync()` method is a non-blocking alternative to `Atomics.wait()`.
```